kernel/src/utilities/copy.rs

```diff
@@ -0,0 +1,88 @@
+use crate::ErrorCode;
+use core::ptr;
+
+pub trait CopyOrErr {
+    /// Copies a nonoverlapping slice from src to self. Returns Err(ErrorCode) is source and slef
```

Comment:
```suggestion
    /// Copies a nonoverlapping slice from src to self. Returns Err(ErrorCode) if source and self
```

Reply:
Done along with the rename update in ble_advertising_driver.